From eff02d32807a7d4cac537460aa7b17718227db86 Mon Sep 17 00:00:00 2001
From: Steve Dickson <SteveD@redhat.com>
Date: Thu, 26 Jun 2014 15:54:35 -0400
Subject: [fs] nfs: Fix typo in dprintk

Message-id: <1403798077-28799-77-git-send-email-steved@redhat.com>
Patchwork-id: 84901
O-Subject: [RHEL7.1 PATCH 76/78] NFSv4.1: Fix typo in dprintk
Bugzilla: 1113248
RH-Acked-by: J. Bruce Fields <bfields@redhat.com>

From: Tom Haynes <Thomas.Haynes@primarydata.com>

Signed-off-by: Tom Haynes <Thomas.Haynes@primarydata.com>
Signed-off-by: Trond Myklebust <trond.myklebust@primarydata.com>
Upstream Id: f383b7e8fd77878c24dc682ced9945ab0930ce86
Bugzilla: 1113248
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/nfs/filelayout/filelayout.c b/fs/nfs/filelayout/filelayout.c
index c7fbc91..d2eba1c 100644
--- a/fs/nfs/filelayout/filelayout.c
+++ b/fs/nfs/filelayout/filelayout.c
@@ -279,7 +279,7 @@ filelayout_set_layoutcommit(struct nfs_pgio_data *wdata)
   return;
 
  pnfs_set_layoutcommit(wdata);
- dprintk("%s ionde %lu pls_end_pos %lu\n", __func__, hdr->inode->i_ino,
+ dprintk("%s inode %lu pls_end_pos %lu\n", __func__, hdr->inode->i_ino,
   (unsigned long) NFS_I(hdr->inode)->layout->plh_lwb);
 }
 
-- 
1.7.1